### DISCUSSION TITLE
Jupyter Book build fails after default create

### DISCUSSION THREAD

**Discussion by MantiMantilla** [executablebooks/meta#166] [Q&A] [answered]
I'm still learning how this all works so please bear with me.

I'm running conda 4.8.5 on my Windows 10 machine. I've already installed all necessary Jupyter extensions, I think. (Jupyter Lab, Jupyter Notebook, Jupyter Book, Node.js, and their dependencies).

The problem might have to do with the fact that I've installed Miniconda on a separate (D:/) drive.

I've set up a virtual environment (MyEnv) with all the packages I might need for this project. These are the steps I follow:

- Launch CMD window
- $ conda activate MyEnv
- $ jupyter-lab --notebook-dir "Documents/Jupyter Books"
- At this point a browser tab opens running Jupyter Lab
- From the launcher within Jupyter Lab, open a terminal
- $ cd "Documents/Jupyter Books"
- $ jb create MyCoolBook
- New folder with template book contents gets created in this directory (Yay!)
- Without editing anything: $ jb build MyCoolBook
- A folder gets added to MyCoolBook called _build, but it doesn't contain much more than a few CSS files.
- The terminal throws [this](https://pastebin.com/ygiWA7QW) error traceback which wasn't very helpful to me. The issue may be obvious to an experienced user.

I'm really at my wit's end. I've reset the entire environment a few times trying to get this to work. What do you suggest? A big thank you in advance. I'm considering submitting a bug report but I want to rule out the very reasonable possibility that I'm being silly.

**Comment by chrisjsewell** [comment, marked as answer]
Please see https://jupyterbook.org/advanced/advanced.html#working-on-windows and in particular or notes https://github.com/executablebooks/jupyter-book/issues/906

**Reply by MantiMantilla** [reply]
This worked great! I really should take my time the read through the documentation.

**Reply by chrisjsewell** [reply]
haha no problem 😄
